StyleCop.Analyzers/StyleCop.Analyzers.Test/SpacingRules/SA1009UnitTests.cs

```diff
@@ -0,0 +1,278 @@
+﻿namespace StyleCop.Analyzers.Test.SpacingRules
+{
+    using System.Collections.Generic;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Microsoft.CodeAnalysis.Diagnostics;
+    using StyleCop.Analyzers.SpacingRules;
+    using TestHelper;
+    using Xunit;
+
+    /// <summary>
+    /// Unit tests for <see cref="SA1009ClosingParenthesisMustBeSpacedCorrectly"/>
+    /// </summary>
+    public class SA1009UnitTests : CodeFixVerifier
```

Comment:
:bulb: For diagnostics that don't have a code fix, it's best to use `DiagnosticVerifier` as the base class.